catalog/pipeline/fbc-release/0.12/README.md

```diff
@@ -0,0 +1,78 @@
+# FBC Release Pipeline
+
+ Tekton release pipeline to interact with FBC Pipeline                       
```

Comment:
Nit: I just noticed all the extra spaces at the end of the line. Can we remove it? Also the leading space.